addon-test-support/helpers.js

```diff
@@ -1,27 +1,27 @@
 import Ember from 'ember';
 import wait from 'ember-test-helpers/wait';
 
-const { run, $, merge } = Ember;
+const { run, merge } = Ember;
 
 const DEFAULT_EVENT_OPTIONS = { canBubble: true, cancelable: true };
 const KEYBOARD_EVENT_TYPES = ['keydown', 'keypress', 'keyup'];
 const MOUSE_EVENT_TYPES = ['click', 'mousedown', 'mouseup', 'dblclick', 'mouseenter', 'mouseleave', 'mousemove', 'mouseout', 'mouseover'];
 
 function focus(el) {
   if (!el) { return; }
-  let $el = $(el);
-  if ($el.is(':input, [contenteditable=true]')) {
-    let type = $el.prop('type');
+
+  if (el.tagName === 'INPUT' || el.contentEditable || el.tagName === 'ANCHOR') {
+    let type = el.type;
     if (type !== 'checkbox' && type !== 'radio' && type !== 'hidden') {
       run(null, function() {
         // Firefox does not trigger the `focusin` event if the window
-        // does not have focus. If the document doesn't have focus just
-        // use trigger('focusin') instead.
-
-        if (!document.hasFocus || document.hasFocus()) {
-          el.focus();
+        // does not have focus. If the document does not have focus then
+        // fire `focusin` event as well.
+        if (document.hasFocus && !document.hasFocus()) {
```

Comment:
The gotcha with focus is that Firefox won't fire `focus` if the window isn't in focus. (i.e. launched in background with testem). So just fire the event anyway.